config/environment.js

```diff
@@ -58,7 +58,7 @@ module.exports = function(environment) {
       PRELOAD_AUTHORIZED_TYPES: ["order", "gogovan_transport"],
 
       SHA: process.env.APP_SHA || "00000000",
-      VERSION: "1.0.0"
+      VERSION: process.env.VERSION || "1.0.0"
```

Comment:
This will work on the pre-compiled mobile versions but not on the web apps as we run several in the same env. Is there anyway to get this version from the package.json version?